src/Http/Controllers/ResourceController.php

```diff
@@ -24,10 +25,36 @@ public function index(Request $request, $resourceHandle)
             abort('403');
         }
 
+        $columns = $this->buildColumns($resource, $blueprint);
+        
+        $count = $resource->model()
+            ->count();
+
+        return view('runway::index', [
+            'title'    => $resource->name(),
+            'resource' => $resource,
+            'recordCount'  => $count,
+            'columns'  => $columns,
+            'filters'  => Scope::filters($resourceHandle),
+        ]);
+    }
+
+    public function api(FilteredRequest $request, $resourceHandle)
+    {
+        $resource = Runway::findResource($resourceHandle);
+        $blueprint = $resource->blueprint();
+
+        if (! User::current()->hasPermission("View {$resource->plural()}") && ! User::current()->isSuper()) {
+            abort('403');
+        }
+
+        $sortField = request('sort', $resource->listingSort()['column']);
```

Comment:
Instead of using the `request()` helper here, can we use `$request`? Again, another thing that is personal preference.

Reply:
Yep! Done.